src/components/Form/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect }from 'react';
+import { useDispatch } from 'react-redux';
+
+import { getResult } from '../../redux/actions'
+
+
+const Form = () => {
+
+    const dispatch = useDispatch();
+    const [nameInput, setNameInput] = useState("")
+
+    const handleSubmit = (e) => {
+        e.preventDefault()
+        dispatch(getResult(nameInput))
+        setNameInput("")
+    }
+
+    const updateInput = e => {        
+        setNameInput(e.target.value);
+    }
+    
+    return (
+        <form onSubmit={handleSubmit} role= "form">
```

Comment:
Only picking this up because I did the exact same thing in my assignment  <form role="form>. According to MDN, "You do not need to declare role="form" on every form element (inputs, text areas, selects, etc.). It should be declared on the HTML element that wraps the form elements. Ideally, use the <form> element as the wrapping element and do not declare role="form".

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Form_Role

Reply:
Oh wow, that's great to know!  thanks for this.  :)